source/Weapon.h

```diff
@@ -57,6 +57,7 @@ class Weapon {
 
 
 public:
+	Weapon() {}
```

Comment:
Any reason why you had to add this, and why it isn't `= default`, `noexcept`, etc.?

Reply:
changed